src/spyglass/spikesorting/analysis/v1/group.py

```diff
@@ -132,6 +132,9 @@ def fetch_spike_data(
             dictionary containing the group key
         time_slice : list of float, optional
             if provided, filter for spikes occurring in the interval [start, stop], by default None
+        return_unit_ids : bool, optional
+            if True, return the unit_ids along with the spike times, by default False
+            Unit ids defined as "{merge_id}_{unit_number}"
```

Comment:
Suggested type change

```suggestion
            Unit ids defined as Tuple[merge_id, unit_number}]
```

Reply:
I would rather return a dictionary: `{"spikesorting_merge_id":merge_id, "unit_id":unit_id}` to avoid concatenation, since the dictionary is more directly usable than a tuple (e.g. to insert into a table).

Would that still resolve the concatenation concerns?

Reply:
Sure, that works for me! I have a slight preference for funcs that return either (a) a tuple of similar types, or (b) a single dict of all return values. I avoid returning one thing and then a dict of the others, but that's stylistic